sklearn/utils/testing.py

```diff
@@ -47,6 +54,8 @@
 from numpy.testing import assert_array_less
 import numpy as np
 
+from contextlib import contextmanager
```

Comment:
This import is not needed. Unless you rewrite your context manager to use the `yield` syntax trick.